theme/ckeditor5-mention/mentionediting.css

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-color-mention-background: hsla(341, 100%, 30%, 0.1);
+	--ck-color-mention-text: hsl(341, 100%, 30%);
+}
+
+.mention {
```

Comment:
We need here 020 CSS Specificity - `.ck .mention`.

Reply:
`.ck-content .mention` as per slack suggestion or `.ck .mention` ?

Reply:
Let's go with `.ck-content .mention` 👍